### PR TITLE
Submit DevToysMac 0.0.7.2

### PR DESCRIPTION
**Pull request by SeanSith**
Submitting DevToysMac 0.0.7.2

### CODE REVIEW

File: Casks/devtoysmac.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask "devtoysmac" do
```
Comment (bevanjkay) [inline Casks/devtoysmac.rb:1]: @SeanSith The token should be `devtoys` and the filenames `devtoys.rb` to match.

File: Casks/devtoysmac.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  version "0.0.7.2"
+  sha256 "e8cd5a2c2670fcf47d86e3bb19de7f9fa853c79ca245f09dd3fa4d9895d8a451"
+
+  url "https://github.com/ObuchiYuki/DevToysMac/releases/download/#{version}/DevToys.app.zip"
+  name "DevToysMac"
+  desc "Swiss Army knife DevToys"
```
Comment (bevanjkay) [inline Casks/devtoysmac.rb:7]: @SeanSith The description should describe what the app does - https://docs.brew.sh/Cask-Cookbook#stanza-desc

File: Casks/devtoysmac.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+cask "devtoysmac" do
+  version "0.0.7.2"
+  sha256 "e8cd5a2c2670fcf47d86e3bb19de7f9fa853c79ca245f09dd3fa4d9895d8a451"
+
+  url "https://github.com/ObuchiYuki/DevToysMac/releases/download/#{version}/DevToys.app.zip"
```
Comment (bevanjkay) [inline Casks/devtoysmac.rb:5]: @SeanSith At present the GitHub repo is too new. We will need to wait until it is 30 days old for it to pass the CI requirements.

**Review by bevanjkay — DISMISSED** [review]

File: Casks/devtoysmac.rb (hunk 1 above)
Comment (SeanSith) [reply Casks/devtoysmac.rb:1]: I didn't name the product, just someone interested in it being available through Homebrew Cask. Maybe this is a non-starter to have it be added?

Comment (bevanjkay) [reply Casks/devtoysmac.rb:1]: @SeanSith The [linked documentation](https://docs.brew.sh/Cask-Cookbook#token-reference) explains the `token reference` guidelines.

**Comment by SeanSith** [comment]
Changes to the name and description addressed.

**Comment by miccal** [comment]
Thank you @SeanSith.

I am happy to disregard the audit failure regarding the age of the GitHub repository considering its number of forks and stars.

**Pull request merged**